OpenRA.Mods.Common/Lint/LintExts.cs

```diff
@@ -0,0 +1,33 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Reflection;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Lint
+{
+	public class LintExts
+	{
+		public static string[] GetFieldValues(ITraitInfo traitInfo, FieldInfo fieldInfo, Action<string> emitError)
```

Comment:
It doesn't look like you depend on anything `ITraitInfo` specific in here, so maybe change this to `object`?